inst/www/shiny.router.js

```diff
@@ -21,12 +21,13 @@ window.shinyrouter = function() {
         var newhash = false;
         var oldhash = window.location.hash;
         if (arguments.length >= 3 && typeof arguments[2] === 'string') {
-            // The easiest way to parse out the hash component (if any) of
-            // the new URL, is to use a browser native "URL" object.
-            var u = new URL(arguments[2], "https://example.com");
-            newhash = u.hash;
+            var newhash = "";
+            var url = arguments[2];
```

Comment:
Technically this is not an url, let's change the variable name to avoid misunderstandings.
Let's name it `path_params_and_anchor`

To clarify:
`url = {protocol}://{hostname}:{port}/{path}?{params}#{anchor}`